recipes/yarn.rb

```diff
@@ -153,22 +153,14 @@
   end
 
   hopsUtil=File.basename(node['hadoop_spark']['hopsutil']['url'])
-  remote_file "#{Chef::Config['file_cache_path']}/#{hopsUtil}" do
+  remote_file "#{node['hadoop_spark']['home']}/jars/#{hopsUtil}" do
```

Comment:
This is not going to work. This part of the code is executed only on the Hopsworks node. (Only one node needs to put stuff on HopsFS).
While it's ok to remove the `hops_hdfs_directory` below, the download of the hopsutil library needs to be moved out of the if, so that it's executed on all the machines. 

Also since you are here, could you please remove the part uploading the pyspark archives? it's a left over from the previous PR.

Reply:
Done